### ISSUE TITLE
ElmerComponent stranded, resistance is not set

### ISSUE THREAD

**Issue by OrionFH** [ElmerCSC/elmer_circuitbuilder#3]
When using an ElmerComponent inside of a circuit, its resistance value is not set correctly. Inside the generated circuit file it is always zero

**Comment by jvela018** [comment]
@OrionFH ,

Yes, I have initially set it as 0. As you need to enter the DC resistance of the wire, I thought it'd be easier to let the user enter that variable manually. Since in this case you'd need the Area, resistivity/Conductivity, I was (am) not completely sure how this is handled from within Elmer.  I'll keep this issue open until I have time to work on an alternative.

BR,

/Jon

**Comment by jvela018** [comment]
@OrionFH ,

I have yet to update it to PyPi, but I've created a patch for the resistance issue. It is included in PR #8 . It's been merged to main and can be used directly through the repository. I'll package it and update it on PyPi as soon as I get a change. 

/Jon

**Comment by jvela018** [comment]
@raback , This issue can be closed. This issue is addressed in PR #8
